apps/common-app/src/examples/LayoutAnimations/LayoutTransitionExample.tsx

```diff
@@ -0,0 +1,295 @@
+            transition:
+              EntryExitTransition.entering(FlipInEasyX).exiting(FlipOutYRight),
+          })
+        }
+      />
+    </View>
+  );
+}
+
+const Stack = createStackNavigator<RootStackParamList>();
+
+export default function Layout() {
+  return (
+    <NavigationContainer independent={true}>
+      <Stack.Navigator screenOptions={{ title: 'Layout Transitions 🔥' }}>
+        <Stack.Screen name="Home" component={HomeScreen} />
+        <Stack.Screen
+          name="linear"
+          component={Transition}
+          initialParams={{
+            title: 'LinearTransition',
+            transition: LinearTransition,
+          }}
+        />
+        <Stack.Screen
+          name="fading"
+          component={Transition}
+          initialParams={{
+            title: 'FadingTransition',
+            transition: FadingTransition,
+          }}
+        />
+        <Stack.Screen
+          name="sequenced"
+          component={Transition}
+          initialParams={{
+            title: 'SequencedTransition',
+            transition: SequencedTransition,
+          }}
+        />
+        <Stack.Screen
+          name="jumping"
+          component={Transition}
+          initialParams={{
+            title: 'JumpingTransition',
+            transition: JumpingTransition,
+          }}
+        />
+        <Stack.Screen
+          name="curved"
+          component={Transition}
+          initialParams={{
+            title: 'CurvedTransition',
+            transition: CurvedTransition,
+          }}
+        />
+        <Stack.Screen
+          name="entryexit"
+          component={Transition}
+          initialParams={{
+            title: 'EntryExit',
+            transition:
+              EntryExitTransition.entering(FlipInEasyX).exiting(FlipOutYRight),
+          }}
+        />
+      </Stack.Navigator>
+    </NavigationContainer>
+  );
+}
+
+type TransitionProps = StackScreenProps<
+  RootStackParamList,
+  'linear' | 'fading' | 'sequenced' | 'jumping' | 'curved' | 'entryexit'
+>;
+
+function Transition({ route }: TransitionProps) {
+  const { title, transition } = route.params;
+  const [items, setItems] = useState(INITIAL_LIST);
+  const [selected, _] = useState({ label: title, value: transition });
+
+  const removeItem = (idToRemove: number) => {
+    const updatedItems = items.filter((item) => item.id !== idToRemove);
+    setItems(updatedItems);
+  };
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <Text style={styles.title}>{title}</Text>
+      <View>
+        <Items selected={selected} items={items} onRemove={removeItem} />
+      </View>
+    </SafeAreaView>
+  );
+}
+
+interface ItemsProps {
+  selected: { label: string; value: any };
+  items: Item[];
+  onRemove: (id: number) => void;
+}
+
+function Items({ selected, items, onRemove }: ItemsProps) {
+  return (
+    <View style={styles.gridContainer}>
+      {items.map((item: { id: number; content: string; color: string }) => (
```

Comment:
Can we just do in this way? Because you have already defined `Item` type.
```suggestion
      {items.map((item: Item) => (
```